lib/ui/common/CircularProgress.dart

```diff
@@ -91,6 +17,11 @@ class CircularProgress extends StatefulWidget {
 class _CircularProgressState extends State<CircularProgress> {
   double defaultValue = 0;
   double percentageComplete = 100.0;
```

Comment:
try and use normalised terms (0.0-1.0). percent is a display implementation detail.